python/ray/tests/test_autoscaler.py

```diff
@@ -3620,7 +3620,9 @@ def testScaleDownIdleTimeOut(self):
         worker_ip = self.provider.non_terminated_node_ips(WORKER_FILTER)[0]
         # Mark the node as idle
         lm.update(worker_ip, mock_raylet_id(), {"CPU": 1}, {"CPU": 1}, 20)
+        assert lm.is_active(worker_ip)
         autoscaler.update()
+        assert not lm.is_active(worker_ip)
```

Comment:
`lm.is_active(worker_ip)` should still be `True` before `autoscaler.update()`.
`lm.is_active(worker_ip)` should be `False` after `autoscaler.update()` because the idle `worker_ip` should be pruned by the change in this PR.